project/src/main/java/com/google/sps/data/Event.java

```diff
@@ -0,0 +1,205 @@
+package com.google.sps.data;
+
+import java.lang.StringBuilder;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Random; 
+import java.util.Set;
+
+/** Class containing Event object 
+Setters for variables that user can change about event
+*/
+public final class Event {
+  private final int eventId = 1000000;
```

Comment:
I'm using primitive longs for IDs (and arbitrarily made them length 8), but I can't find any general consensus on object ID fields. @liangqigong @hlz03 does Google have guidelines?

Reply:
I don't find any guideline, but yes we usually use long for ids.

Reply:
done